hack/update-generated-kms.sh

```diff
@@ -25,5 +25,6 @@ KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 # If the output set is changed update that function.
 
 "${KUBE_ROOT}/build/run.sh" hack/update-generated-kms-dockerized.sh "$@"
+APIROOTS="k8s.io/apiserver/pkg/storage/value/encrypt/envelope/kmsv2/v2alpha1" hack/update-generated-protobuf.sh "$@"
```

Comment:
This only takes care of v2alpha1 right? what if there are changes to v1 in the future?

Reply:
This would be similar to how we generate the proto files for the KMS gRPC API - https://github.com/kubernetes/kubernetes/blob/f88f76f4448d83fd5b825aa6722724f68905738e/hack/update-generated-kms-dockerized.sh#L22-L23

For the newer version, we'll add comma separated packages to generate code